### PR TITLE
feat: unit tests for embed pages

### LINKED ISSUE #9386: [CAL-1901] Unit Test: window.isEmbed()
Avoids this bug, https://github.com/calcom/cal.com/pull/9385

Check the manual tests done in the PR and add unit tests for those 
Screenshot 2023-06-07 at 2 08 30 PM

e.g. set window.name correctly and verify that isEmbed returns true

<sub>[CAL-1901](https://linear.app/calcom/issue/CAL-1901/unit-test-windowisembed)</sub>

### PR DESCRIPTION
**Pull request by Pradumn27**
## What does this PR do?

This PR adds e2e tests to check for the styling of the embed page of eventType and also to check whether isEmbed becomes true once we change the window.name property

Fixes #9386 
/claim #9386 

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [x] Run automated e2e tests

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: apps/web/playwright/embed-pages.e2e.ts
Comment (hariombalhara) [inline apps/web/playwright/embed-pages.e2e.ts:1]: @Pradumn27 Let's move this file along with this file in the same folder https://github.com/calcom/cal.com/blob/3f93e6000d9285486a62f9d55bdb3d726d9f5882/packages/embeds/embed-core/playwright/tests/action-based.e2e.ts#L17

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Requested one change @Pradumn27 . If tests pass after that it will be good to merge.

**Comment by Pradumn27** [comment]
> Requested one change @Pradumn27 . If tests pass after that it will be good to merge.

Thanks for the review @hariombalhara , have changed the file location and the tests are also passing!

**Comment by Pradumn27** [comment]
> Requested one change @Pradumn27 . If tests pass after that it will be good to merge.

Thanks for the review @hariombalhara ,I have done the requested changes and the tests are passing!

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**